examples/createSegmentationRLERoundtrip/index.html

```diff
@@ -205,7 +204,7 @@ <h1>Create DICOM Segmentation IOD</h1>
         const t0 = performance.now();
 
         const {
-          labelmapBuffer,
+          labelmapBufferArray,
```

Comment:
why `BufferArray`?  Isn't this an `ArrayBuffer`?

Reply:
no now it's an array of buffers (N elements = N of overlapping segmentations). In the case there are no overlapping segmentation it is still an array of 1 element. We could rename the variable names if you think is not clear enough (e.g.: labelMapsBuffer / labelMapsSeriesBuffer/....?)  .

Reply:
Ah, in that case I'd probably use `labelmapBuffers` (I find it's readable to have something like `buffers.forEach( (buffer) => {} );`)